src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+function successHandler(message) {
+  const div = document.createElement('div');
+
+  div.className = 'message';
+  div.textContent = message;
+
+  document.body.appendChild(div);
+}
+
+function errorHandler(message) {
+  const div = document.createElement('div');
+
+  div.className = 'message error-message';
+  div.textContent = message;
+
+  document.body.appendChild(div);
+}
```

Comment:
In the `errorHandler` function, the `message` parameter is expected to be a string, but when handling promise rejections, the `catch` block receives an `Error` object. To display the error message correctly, you should use `message.message` instead of `message` in `div.textContent = message;`.